pytest_asdf/plugin.py

```diff
@@ -16,6 +17,12 @@ def pytest_addoption(parser):
         "asdf_schema_root", "Root path indicating where schemas are stored")
     parser.addini(
         "asdf_schema_skip_names", "Base names of files to skip in schema tests")
+    parser.addini(
```

Comment:
Added a feature here that allows individual tests to be skipped or xfailed via config options